src/containers/ProjectDashboardContainer.jsx

```diff
@@ -20,18 +22,57 @@ const propTypes = {
 };
 
 class ProjectDashboardContainer extends Component {
-
+  constructor() {
+    super();
+    this.handleSearch = this.handleSearch.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+    this.state = {
+      searchText: '',
+      translationLanguage: '',
+    };
+  }
   componentDidMount() {
     const { actions } = this.props;
     actions.fetchProject(this.props.params.project_id);
   }
 
+  handleSearch(event) {
+    this.setState({
+      searchText: event.target.value,
+    });
+  }
```

Comment:
Good point, it will. Question: how do we handle the case where there are more than one translation in a given language? I haven't looked at the code for how Translator do this, but I remember seeing a label with the translator's name to differentiate them.